avm/ptn/deployment-script/import-image-to-acr/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -54,6 +54,7 @@ module testDeployment '../../../main.bicep' = [
       location: resourceLocation
       acrName: dependencies.outputs.acrName
       image: 'mcr.microsoft.com/k8se/quickstart-jobs:latest'
+      newImageName: 'your-image-name:tag'
```

Comment:
Required to set for WAF-alignment?

Reply:
Well, I would expect that in many times a different image is in production, as development of the container and creating the IaC scripts will happen in parallel. Therefore, I set it here.
But to answer your question based on facts from WAF, ... I can't ¯\\_(ツ)_/¯

Reply:
Hey @ReneHezser,
the point is that we try to keep tests 'on-point' as they simultaniously represent examples. For `max` & `defaults` it may be obvious, but for scenario-specific test cases (e.g., waf-aligned, private networking, encryption, etc.) it makes sense to only include the properties needed for the specific scenario to not give the wrong impression that xzy parameter **must** be set for WAF-alignment (if that makes sense).

Reply:
I agree, updated the test and ran set-avmmodule again. Ready to go from my end.